GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -156,11 +156,13 @@ class ChatView: EngagementView {
         guard let rows = numberOfRows?(section) else { return }
```

Comment:
perhaps it is overhead, however, I'd like to propose to add to `guard` one more statement
```swift
guard 
    let rows = numberOfRows?(section),
    rows > count
else { return }
```
to avoid `out of range` exception, just in case.

Reply:
Makes sense, but with `rows >= count`, for cases if it's the first item

Reply:
d91e38d332b823cb0ee0785fa35eb9a4c5f7fc1c